src/main/kotlin/lotto/model/Tickets.kt

```diff
@@ -2,17 +2,23 @@ package lotto.model
 
 class Tickets(private val ticketList: List<Ticket>) {
     companion object {
-        fun generate(amount: Int): Tickets {
-            val ticketCount = amount / Const.PRICE
-            val list =
-                List(ticketCount) {
+        fun generate(
+            amount: Int,
+            manualTickets: List<Numbers> = emptyList(),
```

Comment:
```suggestion
            manualTicketNumbers: List<Numbers> = emptyList(),
```

I think it just represents numbers, so I'd recommend renaming the variable.